tutorial_github_causify_style/Master_GitHub_analysis.py

```diff
@@ -95,24 +93,33 @@
 # %%
 # Choose your users and repositories
 users = [
-    "Shaunak01", "tkpratardan", "Prahar08modi","sandeepthalapanane","indrayudd","Swapnika29","mongolianjesus","cma0416"
+    "Shaunak01",
+    "tkpratardan",
+    "Prahar08modi",
+    "sandeepthalapanane",
+    "indrayudd",
+    "Swapnika29",
+    "mongolianjesus",
+    "cma0416",
 ]
 repos = ["helpers", "tutorials", "cmamp"]
 org = "causify-ai"
 
 # %% [markdown]
 # <a name='pre-feth-all-the-data-you-need-in-cache'></a>
-# ## Pre-feth all the data you need in cache 
+# ## Pre-feth all the data you need in cache
 
 # %% [markdown]
-# Query extraction takes time, so prefetch all data in cache for all the users, repos and time frames you need. once in cache there are several utility functions to help understand the user contribution. Following is the data we will fetch for users in multiple repos for the given period 
+# Query extraction takes time, so prefetch all data in cache for all the users, repos and time frames you need. once in cache there are several utility functions to help understand the user contribution. Following is the data we will fetch for users in multiple repos for the given period
```

Comment:
Ok, but in general it's best to put as much code as possible in markdown and code, rather than notebooks.

Notebooks are kind of ephemeral